supertokens_python/recipe/session/with_jwt/recipe_implementation.py

```diff
@@ -118,27 +123,26 @@ async def refresh_session(
             user_context=user_context,
         )
 
-        await new_session.update_access_token_payload(access_token_payload)
+        await new_session.update_access_token_payload(
+            access_token_payload, user_context
+        )
         return get_session_with_jwt(new_session, openid_recipe_implementation)
 
     og_update_access_token_payload = original_implementation.update_access_token_payload
 
-    async def update_access_token_payload(
-        session_handle: str,
+    async def jwt_aware_update_access_token_payload(
+        session_information: SessionInformationResult,
         new_access_token_payload: Dict[str, Any],
         user_context: Dict[str, Any],
     ) -> bool:
-        session_information = await original_implementation.get_session_information(
-            session_handle, user_context
-        )
-        if session_information is None:
-            return False
-
+        """DEPRECATED: Use merge_into_access_token_payload instead"""
```

Comment:
This deprecated thing should go in the update_access_token_payload function?

Reply:
Done.